lib/ex_aws/operation/rest_query.ex

```diff
@@ -3,37 +3,24 @@ defmodule ExAws.Operation.RestQuery do
     http_method: nil,
     path: "/",
     params: %{},
+    body: "",
     service: nil,
+    action: nil,
+    parser: &ExAws.Utils.identity/2,
   ]
 
   @type t :: %__MODULE__{}
 end
 
 defimpl ExAws.Operation, for: ExAws.Operation.RestQuery do
   def perform(operation, config) do
-    query = operation.params
-    |> URI.encode_query
-
-    headers = [
-      {"content-type", "application/x-www-form-urlencoded"},
-    ]
-
-    ExAws.Request.request(operation.http_method, url(config, operation.path), query, headers, config, operation.service)
+    headers = []
```

Comment:
it became more complicated as headers vary depending on request method and body type. It works works for all current use cases without headers. So I decided to skip it to avoid bloating PR and make a separate PR to set correct headers.